lldb/test/API/commands/protocol/TestMCPUnixSocket.py

```diff
@@ -0,0 +1,23 @@
+import os
+import tempfile
+
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import *
+
+
+class MCPUnixSocketCommandTestCase(TestBase):
+    @skipIfWindows
+    @no_debug_info_test
+    def test_unix_socket(self):
+        """
+        Test if we can start an MCP protocol-server accepting unix sockets
+        """
+
+        temp_directory = tempfile.TemporaryDirectory()
```

Comment:
One note, for unix sockets there is often a character limit of around 104~108 chars depending on the platform.

This will often fetch a directory using TEMP_DIR, which may be overridden by say a CI system to something pretty long.

We may either want to skip the test if the tempdir is to long or try to find a shorter directory like "/tmp", if it exists.